File: TugasPraktikum04/Bahasa Python/PRAK404-2310817320013-AikoAnatashaWendiono.py
while True:
    print("Pilih program")
    print("1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
    opsi = int(input("Masukkan Pilihan: "))

    if 1 <= opsi <= 4:
        angka1 = int(input("Masukkan nilai pertama: "))
        angka2 = int(input("Masukkan nilai kedua: "))

        if opsi == 1:
            hasil = angka1 + angka2
            print(f"Hasil penjumlahan antara {angka1}.00 dengan {angka2}.00 adalah {hasil:.2f}\n")
        elif opsi == 2:
            hasil = angka1 - angka2
            print(f"Hasil pengurangan antara {angka1}.00 dengan {angka2}.00 adalah {hasil:.2f}\n")
        elif opsi == 3:
            hasil = angka1 * angka2
            print(f"Hasil perkalian antara {angka1}.00 dengan {angka2}.00 adalah {hasil:.2f}\n")
        elif opsi == 4:
            if angka2 != 0:
                hasil = angka1 / angka2
                print(f"Hasil pembagian antara {angka1}.00 dengan {angka2}.00 adalah {hasil:.2f}\n")
            else:
                print("Input anda salah, bilangan tidak bisa dibagi\n")
                continue
    elif opsi == 5:
        print("Terimakasih, telah menggunakan kalkulator Aiko Anatasha Wendiono\n\n")
        break
    else:
        print("Input anda salah, silahkan coba lagi\n\n")

File: TugasPraktikum04/Bahasa C/PRAK404-2310817320013-AikoAnatashaWendiono.c
#include<stdio.h>

int main() {
    int opsi, angka1, angka2;
    float hasil;

    do {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &opsi);

        if (opsi >= 1 && opsi <= 4) {
            printf("Masukkan nilai pertama : ");
            scanf("%d", &angka1);
            printf("Masukkan nilai kedua : ");
            scanf("%d", &angka2);

            switch (opsi) {
                case 1:
                    hasil = angka1 + angka2;
                    printf("Hasil penjumlahan antara %d.00 dengan %d.00 adalah %.2f\n\n", angka1, angka2, hasil);
                    break;

                case 2:
                    hasil = angka1 - angka2;
                    printf("Hasil pengurangan antara %d.00 dengan %d.00 adalah %.2f\n\n", angka1, angka2, hasil);
                    break;

                case 3:
                    hasil = angka1 * angka2;
                    printf("Hasil perkalian antara %d.00 dengan %d.00 adalah %.2f\n\n", angka1, angka2, hasil);
                    break;

                case 4:
                    if (angka2 != 0) {
                        hasil = (float)angka1 / angka2;
                        printf("Hasil pembagian antara %d.00 dengan %d.00 adalah %.2f\n\n", angka1, angka2, hasil);
                    } else {
                        printf("Input anda salah, bilangan tidak bisa dibagi\n\n");
                        continue; 
                    }
                    break;
            }
        } else if (opsi == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Aiko Anatasha Wendiono\n\n");
        } else {
            printf("Input anda salah, silahkan coba lagi\n\n");
            continue;  
        }

    } while (opsi != 5);

    return 0;
}

